### PR TITLE
{Compute} `az vmss rolling-upgrade`, `vm extension image` and `vm host`: Migrate commands to CodeGen V2

### PR DESCRIPTION
**Pull request by yanzhudd**
**Related command**

`az vmss rolling-upgrade cancel/start/get-latest`
`az vm extension image list-versions/show/list-names`
`az vm host list/show/delete/restart`

**Description**

Migrate all commands in `az vmss rolling-upgrade` group.

Migrate a part of commands of `az vm extension image` and `az vm host` group.

Some commands in these groups are **not** migrated yet:

- `az vm extension image list`

- `az vm host get-instance-view/create/update`

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Compute

File: src/azure-cli/azure/cli/command_modules/vm/aaz/2018_03_01_hybrid/vmss/rolling_upgrade/_start.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "vmss rolling-upgrade start",
+)
+class Start(AAZCommand):
+    """Starts a rolling upgrade to move all virtual machine scale set instances to the latest available Platform Image OS version. Instances which are already running the latest available OS version are not affected.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/aaz/2018_03_01_hybrid/vmss/rolling_upgrade/_start.py:18]:
```suggestion
    """Start a rolling upgrade to move all virtual machine scale set instances to the latest available Platform Image OS version. Instances which are already running the latest available OS version are not affected.
```

File: src/azure-cli/azure/cli/command_modules/vm/aaz/2019_03_01_hybrid/vmss/rolling_upgrade/_cancel.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "vmss rolling-upgrade cancel",
+)
+class Cancel(AAZCommand):
+    """Cancels the current virtual machine scale set rolling upgrade.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/aaz/2019_03_01_hybrid/vmss/rolling_upgrade/_cancel.py:18]: Please use the first-person voice in the help message

File: src/azure-cli/azure/cli/command_modules/vm/aaz/2019_03_01_hybrid/vmss/rolling_upgrade/_start.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "vmss rolling-upgrade start",
+)
+class Start(AAZCommand):
+    """Starts a rolling upgrade to move all virtual machine scale set instances to the latest available Platform Image OS version. Instances which are already running the latest available OS version are not affected.
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/aaz/2019_03_01_hybrid/vmss/rolling_upgrade/_start.py:18]:
```suggestion
    """Start a rolling upgrade to move all virtual machine scale set instances to the latest available Platform Image OS version. Instances which are already running the latest available OS version are not affected.
```

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**